### ISSUE TITLE
Drop in locale and Unsuccessful credit card verification error messages

### ISSUE THREAD

**Issue by chrisribe** [braintree/braintree-web-drop-in#286]
### General information

* SDK/Library version: 1.7.0
* Environment: Sandbox 
* Browser and OS: Chrome 61.0.3163.100 (64-bit)

### Issue description

When setting up the dropin I specify the locale desired (french in my case). But general errors like credit card verification failed messages (ex: code 81736) display in English.
	braintree.dropin.create({
		locale: 'fr_FR',
		authorization: ctoken,
		container: $dIn.get(0)
	}, 
	function(err, instance) {
	   if (err) {
		return;//errors here 
	    }
	});

![french-english-display](https://user-images.githubusercontent.com/1999791/31028406-d1918afe-a51b-11e7-890f-626bc049db5f.PNG)

Any way to override / fix this ??
Thanks
Christian

**Comment by crookedneighbor** [comment]
Thanks for the report. We'll look into this.

**Comment by lilaconlee** [comment]
Looks like we've got a handful of error translations missing. Thanks for reporting this and we'll get them in as soon as possible!

**Comment by chrisribe** [comment]
Hi, any updates ?

**Comment by lilaconlee** [comment]
We're still waiting on the CVV verification error translations, but if you would like to add your own translations for the missing errors in the meantime you can by including custom `translations` in your create call:
```javascript
braintree.dropin.create({
  authorization: CLIENT_AUTHORIZATION,
  locale: 'fr_FR',
  translations: {
    hostedFieldsTokenizationNetworkErrorError: 'Erreur réseau. Réessayez.',
    paypalAccountTokenizationFailedError: "Une erreur s'est produite au cours de l'enregistrement du compte PayPal. Veuillez réessayer.",
    hostedFieldsTokenizationCvvVerificationFailedError: "Your translation for CVV verification failure"
  // Other configuration options
}, callback);
```

Here is a [codepen](https://codepen.io/lilaconlee/pen/eGQwbv) with that fix. 

We'll let you know when we've released a version with all of the missing translations!

**Comment by stodorovic** [comment]
We had to include some custom translations because we founded inconsistency and possible wrong translations into braintree js file.

For example your translation file ([fr_FR.js](https://github.com/braintree/braintree-web-drop-in/blob/master/src/translations/fr_FR.js)) bring :
````
"payingWith": "Payer avec {{paymentSource}}",
"payWithCard": "Payer par carte",
```

To be understandable to the french audience we added:
```
translations: {
  payingWith: 'Payer par {{paymentSource}}',
  chooseAnotherWayToPay: 'Choisissez une autre mode de paiement.',
  hostedFieldsTokenizationCvvVerificationFailedError: 'Vérification de la carte échouée. Merci de vérifier vos informations et d’essayer à nouveau.',
},
```

Please could you review this information in order to provide translations using the same "style".

**Comment by lilaconlee** [comment]
@stodorovic Thanks for the feedback, we'll take another look at our translations!

We're going to close this issue once we've added the missing error strings, but if you'd like to open a new issue about reviewing our translations we can update you on our progress there.

**Comment by crookedneighbor** [comment]
This is fixed in v1.8.1 of Drop-in

**Comment by crookedneighbor** [comment]
@stodorovic Feel free to open a separate issue for the concern you brought up.

**Comment by Vova11** [comment]
Hi,

is there any chance to load local based on I18n.locale. Something like:
braintree.dropin.create({
    authorization: client_token,
    if (I18n.locale == :en)  {
      locale: 'fr_FR',
    } else { 
      translations: {
      Card: 'Card',
      fieldEmptyForCvv: 'fil in  CVV.',
      },
    },

Or perhaps even better to load from app/config/locales/branintree.xx.yml

Thanks,
V

**Comment by crookedneighbor** [comment]
@Vova11 Sounds like what you need is fairly specific to your app setup. Please contact our support team so we can get the details on what you need and help with your integration: https://articles.braintreepayments.com/forms/contact
